opensciencegrid/ospool-ap/condor/81-osg-flocking-override.conf

```diff
@@ -0,0 +1,10 @@
+#--- Authentication settings overrides from the flocking config:
+# We want to use token auth instead of GSI or PASSWORD.
+
+SEC_DEFAULT_AUTHENTICATION_METHODS = FS, TOKEN
+SEC_CLIENT_AUTHENTICATION_METHODS = FS, TOKEN
+SEC_DAEMON_AUTHENTICATION_METHODS = TOKEN
+GSI_DAEMON_CERT =
+GSI_DAEMON_KEY =
+ALLOW_NEGOTIATOR_SCHEDD = $(ALLOW_NEGOTIATOR_SCHEDD) condor@$(OSG_FLOCK)
```

Comment:
```suggestion
```

This is not needed since we are not doing flocking. Also, OSG_FLOCK is not defined.

Reply:
Addressed in #32